build/Package.props

```diff
@@ -0,0 +1,21 @@
+<Project>
+  <PropertyGroup>
+    <Authors>Bonsai</Authors>
+    <Copyright>Copyright © Bonsai Foundation CIC and Contributors 2022</Copyright>
+    <PackageProjectUrl>https://github.com/bonsai-rx/zeromq</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/bonsai-rx/zeromq.git</RepositoryUrl>
+    <PackageType>Dependency;BonsaiLibrary</PackageType>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <PackageIcon>icon.png</PackageIcon>
+    <IncludeSymbols>true</IncludeSymbols>
+    <RepositoryType>git</RepositoryType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\LICENSE" PackagePath="/" />
```

Comment:
These items should all use `$(MSBuildThisFileDirectory)` instead of `..\` since the latter is relative to the project file.